CHANGELOG.md

```diff
@@ -1,11 +1,47 @@
 # SAS Viya Monitoring for Kubernetes
 
+## Version 1.1.5 (15FEB22)
+
+* **Overall**
+  * [FIX] Several spurious error messages have been suppressed
```

Comment:
Which "several spurious" error messages?  If this is related to the errexit trap, we identified a single spurious message...which we addressed by fixing the code which was generating an error.

Reply:
I thought there was more than one case where this came up. Could you provide more accurate wording?